theurgy.lic

```diff
@@ -120,10 +355,40 @@ get_from_container('wine')
 fput 'pour wine on altar'
 put_in_container('wine')
 
+
+comm_truff
+
+comm_kert
+
 fput "recite Meraud, power the holy fires that unleash my righteous vengeance;Chadatru, guide my sword to swing in justice;Everild, give me the power to conquer my enemies;Truffenyi, let me not lose sight of compassion and mercy;Else, I will become like those I despise;Urrem'tier, receive into your fetid grasp these wicked souls;May the Tamsine's realms never know their evil ways again;May all the Immortals guide your faithful soldier #{checkname}."
-DRCT.walk_to '5988'
-fput 'meditate'
+pause 1
+
+waitrt?
+
+clean_altar
+waitrt?
+
+if $settings.favor_god
+  DRC.bput("pray #{$settings.favor_god}", "#{$settings.favor_god}")
+  waitrt?
+else
+  DRC.bput('pray meraud', 'meraud')
+  waitrt?
+end  
+
+move 'go path'
+move 'go hole'
+waitrt?
+move 'east'
+move 'go hole'
+move 'out'
+
+DRCT.walk_to('1230')
+if 'but there is nothing in there like that' != DRC.bput("rummage /C parchment my #{$settings.theurgy_supply_container}", 'but there is nothing in there like that', 'pale golden prayer parchment')
+  get_from_container('golden parchment')
+  fput 'invoke my parchment'
```

Comment:
Yeah - the parchment it purchases for you but I will make mention of it.  The carving knife/shaper/prayer chain are tied into the YAML addition immortal_aspect so will mention them in that description.